bindings/java/java/src/main/java/org/ethereum/evmc/EvmcVm.java

```diff
@@ -0,0 +1,113 @@
+package org.ethereum.evmc;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The Java interface to the evm instance.
+ *
+ * <p>Defines the Java methods capable of accessing the evm implementation.
+ */
+public final class EvmcVm {
+
+  static {
+    try {
+      System.load(System.getProperty("user.dir") + "/../c/build/evmc.so");
```

Comment:
Can we use System.load without mentioning the relative path?

Reply:
> System.loadLibrary ?

I chose `System.load` so that the consumer of these bindings doesn't have to fiddle with java.library.path.  They can just provide whatever path they want.  That being said, i will make two methods: `Evmc.loadLibrary` and `Evmc.load`

Reply:
Not sure if that can help, this is how we did it (but we used JNR): https://github.com/apache/incubator-tuweni/blob/master/crypto/src/main/java/org/apache/tuweni/crypto/sodium/Sodium.java#L170